pkg/build/pipelines/npm/install.yaml

```diff
@@ -19,7 +20,7 @@ inputs:
   prefix:
     description: |
       The -prefix argument to pass to npm install; where /bin and /lib will be copied to.
-    default: "${{targets.contextdir}}/usr/"
+    default: "${{targets.contextdir}}/usr/local/"
```

Comment:
Thank you for looking at this but distributions shouldn't use `/usr/local`. Anything we provide should land in `/usr` as we are the vendor. `/usr/local` is reserved for the end user. See FHS guidelines here: https://refspecs.linuxfoundation.org/FHS_3.0/fhs/index.html